stdlib/public/core/BridgeObjectiveC.swift

```diff
@@ -369,7 +369,7 @@ public struct AutoreleasingUnsafeMutablePointer<Pointee /* TODO : class */>
     /// Retrieve the value the pointer points to.
     @_transparent get {
       // We can do a strong load normally.
-      return unsafeBitCast(self, to: UnsafeMutablePointer<Pointee>.self).pointee
+      return UnsafeMutablePointer(mutating: self).pointee
```

Comment:
Do we even need to use a mutable pointer?

Stepping back, is this the right label for converting between AUMP and UMP?